themes/bootstrap3/js/facets.js

```diff
@@ -186,48 +193,60 @@ VuFind.register('sideFacets', function SideFacets() {
     $('.side-facets-container-ajax').each(activateSingleAjaxFacetContainer);
   }
 
-  function facetSessionStorage(e) {
+  function facetSessionStorage(e, data) {
     var source = $('#result0 .hiddenSource').val();
     var id = e.target.id;
     var key = 'sidefacet-' + source + id;
-    if (!sessionStorage.getItem(key)) {
-      sessionStorage.setItem(key, document.getElementById(id).className);
-    } else {
-      sessionStorage.removeItem(key);
-    }
+    sessionStorage.setItem(key, data);
   }
 
   function init() {
     // Display "loading" message after user clicks facet:
     activateFacetBlocking();
 
-    // Side facet status saving
-    $('.facet-group .collapse').each(function openStoredFacets(index, item) {
-      var source = $('#result0 .hiddenSource').val();
-      var storedItem = sessionStorage.getItem('sidefacet-' + source + item.id);
-      if (storedItem) {
-        var saveTransition = $.support.transition;
-        try {
-          $.support.transition = false;
+    // Restore state of collapsed and expanded side facets:
+    var saveTransition = null;
+    // Handle transitions only with Bootstrap 3 (bootstrap is undefined):
```

Comment:
I'm not sure I understand this comment; maybe it's worth fleshing out a little bit -- i.e. We only need to handle transitions for Bootstrap 3 (detected when bootstrap is undefined) because [why?].

Reply:
I rewrote the transition handling and enabled transitions with Bootstrap 5 too.